### PR TITLE
fix usage of BASE_DN env

### LINKED ISSUE #16: BASE_DN is ignored
Commit 9000a6c8 (2019-12-30) introduced a problem during container initialization, starting from line 53 of `/etc/cont-init.d/10-openldap`:

```
  IFS='.' read -a domain_elems <<< "${DOMAIN}"
  SUFFIX=""
  ROOT=""

  for elem in "${domain_elems[@]}" ; do
      if [ "x${SUFFIX}" = x ] ; then
          SUFFIX="dc=${elem}"
          ROOT="${elem}"
      else
          BASE_DN="${SUFFIX},dc=${elem}"
      fi
  done
```

This snippet overwrites the `BASE_DN` environment variable, and that loop does not build it correctly from `DOMAIN` for a domain bigger than two components (e.g. my.example.org will end in `BASE_DN=dc=my,dc=org`).

Bug aside, this image is very handy, thank you for your work.

### PR DESCRIPTION
**Pull request by bfidel**
fixes #16 
add condition to not override BASE_DN if exists, at openldap configuration function.

### CODE REVIEW

File: install/assets/functions/10-openldap [hunk 1]
```diff
@@ -196,18 +196,20 @@ configure_ldap_server() {
...
-
-  for elem in "${domain_elems[@]}" ; do
-    if [ "x${SUFFIX}" = x ] ; then
-        SUFFIX="dc=${elem}"
-        ROOT="${elem}"
-    else
-        BASE_DN="${SUFFIX},dc=${elem}"
-    fi
-  done
+  if [ -z $BASE_DN ] ; then
```
Comment (bmalynovytch) [inline install/assets/functions/10-openldap:199]:
```suggestion
  if [ -z "$BASE_DN" ] ; then
```

File: install/assets/functions/10-openldap [hunk 2]
```diff
@@ -196,18 +196,20 @@ configure_ldap_server() {
...
+    IFS='.' read -a domain_elems <<< "${DOMAIN}"
+    SUFFIX=""
+    ROOT=""
+
+    for elem in "${domain_elems[@]}" ; do
+      if [ "x${SUFFIX}" = x ] ; then
```
Comment (bmalynovytch) [inline install/assets/functions/10-openldap:205]:
```suggestion
      if [ "x${SUFFIX}" = "x" ] ; then
```

**Review by bmalynovytch — APPROVED** [review]
Appart from the tiny suggestions, LGTM.

👍

**Comment by tiredofit** [comment]
Thank you. Looks good to me as well. Will merge and tag a new release momentarily.

**Comment by tiredofit** [comment]
Tagged as `tiredofit/openldap/7.1.2` - Docker Hub should have a build up in the next hour.

**Pull request merged**